janusgraph-server/src/test/java/org/janusgraph/graphdb/tinkerpop/AbstractGremlinServerIntegrationTest.java

```diff
@@ -48,7 +48,7 @@
     private static final Logger logger = LoggerFactory.getLogger(AbstractGremlinServerIntegrationTest.class);
 
     public String getSettingsPath() {
-        return "src/test/resources/gremlin-server-integration.yaml";
+        return "src/test/resources/janusgraph-server-integration.yaml";
```

Comment:
The naming is a little bit unconvenient here: In the integration tests, `gremlin-server-[...].yaml` was renamed to `janusgraph-server-[...].yaml`, while in `janusgraph-dist/`, all config files are still named `gremlin-server-[...].yaml`.

Reply:
Oh right, I thought that they were named `janusgraph-server-[...].yaml` already in the distribution, but then I'll also change the name back here.